app/javascript/src/MappingRules/components/NewMappingRule.jsx

```diff
@@ -0,0 +1,88 @@
+// @flow
+
+import * as React from 'react'
+
+import {
+  ActionGroup,
+  Button,
+  Form,
+  PageSection,
+  PageSectionVariants
+} from '@patternfly/react-core'
+import { CSRFToken } from 'utilities/utils'
```

Comment:
I think you have to update this import
```suggestion
import { CSRFToken } from 'utilities/CSRFToken'
```

Reply:
Good catch! I'll rebase with master and push again